<launch>
  <node name="cam0" pkg="usb_cam" type="usb_cam_node" output="screen" >
      <param name="video_device" value="/dev/video0"></param> 
      <param name="pixel_format" value="yuyv"></param>
      <param name="framerate" value="15"></param> 
      <param name="io_method" value="mmap"></param>
  </node>
  <node name="cam1" pkg="usb_cam" type="usb_cam_node" output="screen" >
      <param name="video_device" value="/dev/video2"></param> 
      <param name="pixel_format" value="yuyv"></param> 
      <param name="framerate" value="15"></param> 
      <param name="io_method" value="mmap"></param>
  </node>
</launch>